### PR TITLE
Some changes in ukrainian translation.

### PR DESCRIPTION
**Pull request by l13t**
@dmytro can you check please this PR? i could be wrong in some moments.

### CODE REVIEW

File: README-uk.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 Ця робота є результатом [багатьох авторів і перекладачів](AUTHORS.md).
 Дещо з цього [спочатку](http://www.quora.com/What-are-some-lesser-known-but-useful-Unix-commands) [появилось](http://www.quora.com/What-are-the-most-useful-Swiss-army-knife-one-liners-on-Unix) на [Quora](http://www.quora.com/What-are-some-time-saving-tips-that-every-Linux-user-should-know), але потім переїхала на Github, де знаходяться дуже талановиті люди які зробили багато удосконалень.
-[**Будь-ласка, зробіть свій внесок**](/CONTRIBUTING.md) якщо Ви замітили помилку або можете щось покращити!
+[**Будь-ласка, зробіть свій внесок**](/CONTRIBUTING.md) якщо Ви помітили помилку або можете щось покращити!
```
Comment (dmytro) [inline README-uk.md]: This piece was already fixed in me PR, but it's not merged in yet.

File: README-uk.md [hunk 2]
```diff
@@ -284,12 +284,12 @@
...
 - `split` і `csplit`: розбити файли
 - `sponge`: прочитати весь інпут перед тим, як його записати. Корисно, коли читаєш з того ж файлу, куди записуєш. Наприклад, ось так: `grep -v something some-file | sponge some-file`
-- `units`: конвертер. Метри в кілометри, версти в п'яді (дивіться `/usr/share/units/definitions.units`)
+- `units`: використовується для конвертації одиниць вимірювання (дивіться `/usr/share/units/definitions.units`)
 - `apg`: генерація паролів
 - `7z`: архіватор з високим ступенем стиснення
 - `ldd`: показує залежності програми від системних бібліотек
 - `nm`: symbols from object files
-- `ab`: бенчмаркінг веб-серверів
+- `ab`: вбудований в Apache бенчмарк веб-серверів
```
Comment (dmytro) [inline README-uk.md]: Hm. It's not actually a built-in in Apache. Maybe something like 'розрозблений командою Apache'?

File: README-uk.md [hunk 3]
```diff
@@ -199,20 +199,20 @@
...
+- Використовуйте `dmesg`, коли щось відбувається зовсім дивне (це можуть бути проблеми із залізом, драйверами, тощо).
 
 
 ## Одним рядком
 
 Кілька прикладів склеювання команд:
-- Це досить круто, що можна знайти множинні перетину файлів, з'єднати відсортовані файли і подивитися різницю в декількох файлів через `sort`/`uniq`. Це швидкий підхід і працює на файлах будь-якого розміру (включаючи багатогігабайтними файли). (Сортування не обмежено пам'яттю, але можливо вам доведеться додати `-T`, якщо `/tmp` знаходиться на невеликому логічному диску). Ще подивіться на те, що було сказано вище про `LC_ALL`. Прапор сортування `-u` не використовується нижче, щоб було зрозуміліше:
+- Це досить круто, що можна знайти множинни перетину файлів, з'єднати відсортовані файли і подивитися різницю декількох файлів через `sort`/`uniq`. Це швидкий підхід і працює на файлах будь-якого розміру (включаючи багатогігабайтні файли). (Сортування не обмежено пам'яттю, але можливо вам доведеться додати `-T`, якщо `/tmp` знаходиться на невеликому логічному диску). Ще подивіться на те, що було сказано вище про `LC_ALL`. Опція сортування `-u` не використовується нижче, щоб було зрозуміліше:
```
Comment (dmytro) [inline README-uk.md]: Мабуть `множини` (Single 'н')?

**Comment by dmytro** [comment]
Hi, @l13t Thanks for the contributing. There are some pending changes already submitted for the merge but, not merged in yet. Let's wait until it's merged and look at this PR later on.

**Comment by dmytro** [comment]
ref. to previous pr: #409

**Comment by l13t** [comment]
@dmytro ok. i'll wait for merge and then 'll update my PR.

**Comment by dmytro** [comment]
@l13t 

I'd like to change a Ukrainian translation workflow a bit. In my fork I've created feature branch https://github.com/dmytro/the-art-of-command-line/tree/feature/uk_translation 

Could you please submit your current PR and future PR's too against this branch. This way we'll have less dependencies on main master branch. I'll take care of submitting of PR's from feature branch to the main tree when ready.

/cc @jlevy 

P.S. If this is for some reason inconvenient or you have a better ideas please speak up.

**Comment by jlevy** [comment]
Thanks @l13t @dmytro 
If you are coordinating closely, sure, just do whatever works best and then tell me when it's ready to merge.

That said, @dmytro I'm not sure it will work to make _everyone_ use your branch in the future, as you'd then always have to keep it merged up with master all the time. The workflow I've found best with other languages so far is if we always allow PRs but I await your review on all uk items.

**Comment by jlevy** [comment]
@l13t @dmytro any updates? If you can resolve the conflicts, review, and update this PR, I'd be glad to merge and get the uk translation updated.

**Comment by dmytro** [comment]
@jlevy Thanks, I've submitted another PR with the resolved conflicts. This can be closed. Ref: #445

**Pull request merged**